packages/client/src/errors/navi-adapter-error.ts

```diff
@@ -2,9 +2,8 @@
  * Copyright 2020, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
-function isPresent<T>(t: T | undefined | null | void): t is T {
-  return t !== undefined && t !== null;
-}
+
+import { isPresent } from '../utils';
```

Comment:
```suggestion
import { isPresent } from '../utils.js';
```

Reply:
I get this when I add the `.js`:
```
src/errors/navi-adapter-error.ts:6:27 - error TS2307: Cannot find module '../utils.js' or its corresponding type declarations.

6 import { isPresent } from '../utils.js';
                            ~~~~~~~~~~~~~


Found 1 error in src/errors/navi-adapter-error.ts:6
```

Reply:
Looks like you have to do `../utils/index.js` :/